pkg/conditions/conditions.go

```diff
@@ -0,0 +1,89 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package conditions
```

Comment:
As far as I know, by contention there needs to be an empty line between a comment and the `package` statement. Otherwise the comment is considered a package comment for the `godocs`.
```suggestion
// SPDX-License-Identifier: Apache-2.0

package conditions
```